packages/dds/tree/src/util/nestedMap.ts

```diff
@@ -41,17 +41,30 @@ export function tryAddToNestedMap<Key1, Key2, Value>(
 
 /**
  * Copies over all entries from the source map into the destination map.
+ * By default, does not override any existing values.
  *
  * @internal
  */
 export function populateNestedMap<Key1, Key2, Value>(
 	source: NestedMap<Key1, Key2, Value>,
 	destination: NestedMap<Key1, Key2, Value>,
+	override = false,
```

Comment:
I think having the parameter is a good idea, in fact, I like it so much that I think it should not come with a default.